TrainingRNExpress/src/app/components/Book/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, Image } from 'react-native';
+
+import { RED, PLACEHOLDER_IMAGE } from '../../../constants';
+
+import styles from './styles.js';
+
+function Book({ image, title, author }) {
+  return (
+    <View style={styles.container}>
+      <Image source={{ uri: image || PLACEHOLDER_IMAGE }} style={styles.icon} borderRadius={25} />
```

Comment:
`PLACEHOLDER_IMAGE` is already being summoned at `defaultProps`, I think this `||` is not necessary 😊

Reply:
The problems is that image can be null

Reply:
My bad !! Forgot about it !!